front-end-web/src/Components/Pages/SelectionPage.tsx

```diff
@@ -1,18 +1,62 @@
+import { Checkbox, List, ListItem, Button } from "@material-ui/core";
 import React, { Fragment } from "react";
 import { Props } from "../../@Types";
+import { getUserCalendars } from "../../scripts";
 
 /**
  * TODO: Add Documentation
  */
-const SelectionPage: React.FunctionComponent<Props> = ({
-  state,
-  handlers,
-  classes
+interface SelectionProps {
+  calendars: calendar[];
+  selectedCalendars: boolean[];
+  handleSelect: Function;
+  handleSubmit: Function;
+  setStage: Function;
+}
+
+const SelectionPage: React.FunctionComponent<SelectionProps> = ({
+  calendars,
+  selectedCalendars,
+  handleSelect,
+  handleSubmit,
+  setStage
 }) => {
   return (
     <Fragment>
       {
-        //TODO
+        <List>
+          {calendars.map((item: calendar, i: number) => {
```

Comment:
Let's switch from `item: calendar` to `item: any` until we can figure out exactly what comes back from the Google API. Also, can you reference `state.calendars` here?